### DISCUSSION TITLE
DO droplet resets custom client deployment

### DISCUSSION THREAD

**Discussion by camelman90** [mozilla/hubs#2746] [General Questions & Discussion] [answered]
When restarting my DO droplet, custom client code gets reset back to the hubs-cloud branch it seems. This means i need to redeploy everything every time it restarts. Is this intended behaviour?

**Comment by camelman90** [comment]
This also happened when I enabled cloudflare workers

**Comment by robinkwilson** [comment, marked as answer]
This issue is logged here please thumbs up it and feel free to track the progress here: https://github.com/mozilla/hubs-cloud/issues/68
